src/main/g8/build.sbt

```diff
@@ -10,7 +10,7 @@ import scala.scalanative.build._
 
 // defaults set with common options shown
 nativeConfig ~= { c =>
-  c.withLTO(LTO.none) // thin
```

Comment:
These were here to help people with these basic settings (using the defaults) with some suggestions in the comments for settings you might want to select - maybe the comments where not explained well but that was deliberate as the first thing people asked for was how to optimize and such.